rebar.config

```diff
@@ -1,7 +1,7 @@
 {erl_opts, [debug_info, {parse_transform, lager_transform}]}.
-{deps, [
-    {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.5.1"}}}
-]}.
+{deps, [{lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.5.1"}}},
+        {meck, ".*", {git, "git://github.com/eproxus/meck.git", {tag, "0.8.6"}}}
```

Comment:
In Makefile I see we are using rebar3. I suggest we use [profiles](http://www.rebar3.org/docs/profiles) so that the test-only dependencies e.g. meck do not end up in the production OTP release.

Reply:
+1